app/assets/stylesheets/_about.scss

```diff
@@ -38,6 +38,30 @@
     max-width: 27em;
   }
 
+  .start-a-chapter {
+    margin-top: 40px;
+
+    .button-wrapper {
```

Comment:
This part is copied from `home.scss`. We might want to make these styles shared so they're not duplicated.